packages/sdk/config/src/loaders/browser.js

```diff
@@ -38,18 +38,21 @@ export const Defaults = () => {
   return __CONFIG_DEFAULTS__;
 };
 
+let localforage = null;
 /**
  * Settings config from browser storage.
  */
 export const Storage = async () => {
   try {
-    const localforage = await import('localforage');
+    if (!localforage) {
+      localforage = await import('localforage');
```

Comment:
My assumption was that once this code path was run once this wouldn't re-load the code but run access the already loaded code. Is that not the case?